### ISSUE TITLE
How to feed auxiliary inputs to my keras model built with functional api in dl4j?

### ISSUE THREAD

**Issue by nivemaj** [deeplearning4j/deeplearning4j#6727]
My model looks like this in Keras:
xInput = Input(shape=(4,),name='Input')
XD1 = Dense(4, activation='relu', kernel_regularizer=regularizers.l2(regularization), name='D1')(xInput)
XB1 = BatchNormalization(name='B1')(XD1)
XD2 = Dense(3, activation='relu', kernel_regularizer=regularizers.l2(regularization), name='D2')(XB1)
XB2 = BatchNormalization(name='B2')(XD2)
auxInput = Input(shape=(1,), name='auxInput')
XConcat = keras.layers.concatenate([XB2,auxInput],name='concat')
XB3 = BatchNormalization(name='B3')(XConcat) 
XD3 = Dense(layer3_depth, activation='relu', kernel_regularizer=regularizers.l2(regularization), name='D3')(XB3)
XB4 = BatchNormalization(name='B4')(XD3)
XOut = Dense(2, activation='softmax', name='Output')(XB4)
model = Model(inputs=[xInput, auxInput], outputs=XOut)

I saved the model & imported this into dl4j
model = KerasModelImport.importKerasModelAndWeights(fullModel);
This line worked.

Now,  **How do I get predictions out of this model in DL4J?**
My keras input looks like this:
[array([0.6098010021932985, 0.6095072055789255, 0.6086789615956749,
        0.964208600572237], dtype=object), array([0.1191916])]

Thanks!!

**Comment by agibsonccc** [comment]
Closing due to inactivity. We cover this already in the examples. If you don't see a response from us, please ask in gitter.
